glpk/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: glpk
+  version: 4.57
+
+source:
+  fn: glpk-4.57.tar.gz
+  url: http://ftp.gnu.org/gnu/glpk/glpk-4.57.tar.gz
+  md5: 237531a54f73155842f8defe51aedb0f
+
+build:
+  features:
+    - vc9      [win and py27]
+    - vc10     [win and py34]
+    - vc14     [win and py35]
```

Comment:
Not inherently... But I have a python package than links against it and I
know there are others as well. And if you build it with vc14 then it can't
be used with python 2.7
On Nov 22, 2015 05:17, "Carlos Cordoba" <notifications@github.com> wrote:

> In glpk/meta.yaml
> <https://github.com/conda/conda-recipes/pull/481#discussion_r45558714>:
>
> > @@ -0,0 +1,23 @@
> > +package:
> > +  name: glpk
> > +  version: 4.57
> > +
> > +source:
> > +  fn: glpk-4.57.tar.gz
> > +  url: http://ftp.gnu.org/gnu/glpk/glpk-4.57.tar.gz
> > +  md5: 237531a54f73155842f8defe51aedb0f
> > +
> > +build:
> > +  features:
> > +    - vc9      [win and py27]
> > +    - vc10     [win and py34]
> > +    - vc14     [win and py35]
>
> I mean, if not, you can select just one vc feature here (for example, vc14)
> instead of listing them all :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conda/conda-recipes/pull/481/files#r45558714>.
>